### ISSUE TITLE
预训练模型的默认学习策略是什么？迁移学习的时候，对于学习策略有什么要求？

### ISSUE THREAD

**Issue by lea4n** [PaddlePaddle/models#3090]
预训练模型的默认学习策略是什么？迁移学习的时候，对于学习策略有什么要求？
lr_strategy: learning rate changing strategy. Default: "piecewise_decay".
pretrained_model: model path for pretraining. Default: None.

**Comment by SunGaofeng** [comment]
请问您这边具体指的是哪个模型的使用？

**Comment by lea4n** [comment]
ResNeXt50_32x4d

**Comment by lea4n** [comment]
> 请问您这边具体指的是哪个模型的使用？

ResNeXt50_32x4d
